tests/gen_cert_test.py

```diff
@@ -181,3 +216,70 @@ def test_cert_date(self, force_date, locale, timezone, expected):
         """
         cert_date = get_cert_date(force_date, locale, timezone)
         assert_equal(expected, cert_date)
+
+
+class CertificateSubtemplateTests(unittest.TestCase):
```

Comment:
w/r/t splitting this into a separate file:
The point isn't that this is presently a massive amount of code, but rather:
- it runs independently of the surrounding suite, so splitting it out makes things a bit more modular.
- that there's a good chance this grows further, given that we've already identified that we want to extend the subtemplate functionality
  - even if _it_ doesn't grow, the others probably will
- The file is already nearing 300 lines and we even added a new class to it mid-PR.
- This is how we ended up with massive 1000+ LOC both here and in platform; everyone kept saying "it's just a few more lines".
- Beyond more focused files, the other benefit to splitting this out now is that we won't need to worry about losing the git history if/when we split it out later.

I could be convinced otherwise, but I guess I'm genuinely curious, what's the argument for grouping them together?